docs/developer_guide.md

```diff
@@ -42,11 +36,11 @@ deploy and port forwarding.
   gcloud auth application-default login
   ```
 
-## Develop mixer locally as a Go server (Recommended)
+## Develop mixer locally as a Go server
 
 **NOTE** This can only develop and test the gRPC server. Since the
```

Comment:
nit: Maybe remove this note all together?

Reply:
Good point. Now I made the entire guide for local development